packages/medusa/src/api/admin/translations/middlewares.ts

```diff
@@ -29,4 +30,14 @@ export const adminTranslationsRoutesMiddlewares: MiddlewareRoute[] = [
     },
     middlewares: [validateAndTransformBody(AdminBatchTranslations)],
   },
+  {
+    method: ["GET"],
+    matcher: "/admin/translations/statistics",
+    middlewares: [validateAndTransformQuery(AdminTranslationStatistics, {})],
+  },
+  {
+    method: ["GET"],
+    matcher: "/admin/translations/translatable-fields",
```

Comment:
suggestion:(non-blocking): do we need the "translatable" prefix here?
```
GET /admin/translations/fields
```

Reply:
I was thinking, maybe it should be settings, and the response has translatable_fields? To prepare the future?